src/test/fixtures/profiles/processed-profile.js

```diff
@@ -495,18 +495,30 @@ function _buildThreadFromTextOnlyStacks(
   return thread;
 }
 
-export function getNetworkMarker(startTime: number, id: number) {
-  const payload: NetworkPayload = {
+export function getNetworkMarkers(startTime: number, id: number) {
+  const name = `Load ${id}: https://mozilla.org`;
+  const startPayload: NetworkPayload = {
     type: 'Network',
     id,
     pri: 0,
-    status: 'STOP',
+    status: 'STATUS_START',
     startTime,
-    endTime: startTime + 1,
+    endTime: startTime + 0.5,
     URI: 'https://mozilla.org',
     RedirectURI: 'https://mozilla.org',
   };
-  return ['Load 123: https://mozilla.org', startTime, payload];
+
+  const stopPayload: NetworkPayload = {
+    ...startPayload,
+    status: 'STATUS_STOP',
+    startTime: startPayload.endTime,
+    endTime: startPayload.endTime + 0.5,
+  };
```

Comment:
Now this function generated a pair of start/stop markers.
Ideally the stop payload should also hold all the possible properties, but I defer this to a future PR.